pkg/auth/handler/api/authenticationflow_v1_create.go

```diff
@@ -15,35 +15,31 @@ import (
 	"github.com/authgear/authgear-server/pkg/util/httproute"
 	"github.com/authgear/authgear-server/pkg/util/httputil"
 	"github.com/authgear/authgear-server/pkg/util/log"
+	"github.com/authgear/authgear-server/pkg/util/slice"
 	"github.com/authgear/authgear-server/pkg/util/validation"
 )
 
 func ConfigureAuthenticationFlowV1CreateRoute(route httproute.Route) httproute.Route {
 	return route.WithMethods("OPTIONS", "POST").WithPathPattern("/api/v1/authentication_flows")
 }
 
-var AuthenticationFlowV1NonRestfulCreateRequestSchema = validation.NewSimpleSchema(`
-{
-	"type": "object",
-	"required": ["type", "name"],
-	"properties": {
-		"type": {
-			"type": "string",
-			"enum": ["signup", "login"]
-		},
-		"name": {
-			"type": "string"
-		},
-		"url_query": { "type": "string" },
-		"batch_input": {
-			"type": "array",
-			"items": {
-				"type": "object"
-			}
-		}
-	}
+var AuthenticationFlowV1NonRestfulCreateRequestSchema *validation.SimpleSchema
+
+func init() {
+	b := validation.SchemaBuilder{}.Type(validation.TypeObject)
```

Comment:
Why we need to construct the schema in init?

Reply:
Because I need to build the schema, which references authflow.AllFlowTypes.